source/api_documentation/network_integration/advertiser_campaigns/_post_advertiser_campaign.rst

```diff
@@ -4,122 +4,24 @@
 
   .. rubric:: Examples
 
```

Comment:
Removed the examples below because they were not really providing anything special over the last remaining example other than the curl command.